.vscode/iTwinUISnippets.code-snippets

```diff
@@ -0,0 +1,39 @@
+{
+  "Separator": {
+    "prefix": "//-",
+    "body": [
+      "// ------------------------------------------------------------------------------------------------",
+      "$1"
+    ],
+    "description": "Creates a separator comment"
+  },
+  "importForwardRef": {
+    "scoope": "javascriptreact,typescriptreact",
+    "prefix": "impforwardRef",
+    "body": [
+      "import * as React from 'react';",
+      "import { Box } from '../utils/index.js';",
+      "import type { PolymorphicForwardRefComponent } from '../utils/index.js';"
+    ],
+    "description": "Imports needed for forwardRef component"
+  },
+  "iTwinUIComponent": {
+    "scope": "typescriptreact",
+    "prefix": "forwardRef",
+    "body": [
+      "const ${1:Name} = React.forwardRef((props, forwardedRef) => {",
+      " const {${2:props}} = props",
+      "",
+      " return (",
+      "   ${3:jsx}",
+      " );",
+      "}) as PolymorphicForwardRefComponent<'${4:element}', ${5:PropsType}>;"
```

Comment:
One last comment: This snippet has too many variables. Can we reduce it to just 1:Name? We can use sensible defaults for the rest (or just omit them):

```jxx
const <CursorGoesHere> = React.forwardRef((props, forwardedRef) => {
 const { ...rest } = props;

 return (
   <Box ref={forwardedRef} {...rest} />
 );
}) as PolymorphicForwardRefComponent<'div'>;
```

Reply:
Okay, will do.

Reply:
Latest commit removes all of the variables but name.